packages/medusa/src/interfaces/payment-service.ts

```diff
@@ -1,18 +1,38 @@
 import { TransactionBaseService } from "./transaction-base-service"
 import {
+  Address,
   Cart,
   Customer,
   Payment,
   PaymentSession,
   PaymentSessionStatus,
+  ShippingMethod,
 } from "../models"
 import { PaymentService } from "medusa-interfaces"
-import { PaymentProviderDataInput } from "../types/payment-collection"
 
 export type Data = Record<string, unknown>
 export type PaymentData = Data
 export type PaymentSessionData = Data
 
+export type PaymentContext = {
+  cart: {
```

Comment:
would make sense to make cart optional so we don't need it when creating a payment collection?

Reply:
same as above